streams/fr.m3u

```diff
@@ -112,10 +112,10 @@ http://188.40.68.167/russia/gulli_girl/playlist.m3u8
 https://dcunilive47-lh.akamaihd.net/i/dclive_1@739146/master.m3u8
 #EXTINF:-1 tvg-id="ILTV.fr" status="error",ILTV (720p) [Not 24/7]
 https://str81.creacast.com/iltv/high/playlist.m3u8
-#EXTINF:-1 tvg-id="KTO.fr" status="error",KTO (404p) [Not 24/7]
-https://livehdkto-lh.akamaihd.net/i/LiveStream_1@178944/master.m3u8
+#EXTINF:-1 tvg-id="KTO.fr" status="online",KTO (404p) [Not 24/7]
+https://live-kto.akamaized.net/hls/live/2033284/KTO/03.m3u8
 #EXTINF:-1 tvg-id="LaChaineNormande.fr" status="error",La Chaîne normande (LCN) (576p) [Not 24/7]
-http://live.lachainenormande.fr/live/lcn/livestream/playlist.m3u8
+https://lcn.live-kd.com/live/lcn/ngrp:lcn_Wifi/playlist.m3u8
```

Comment:
supposed to be https://lcn.live-kd.com/live/lcn/ngrp:lcn_Wifi/manifest.mpd but doesn't work too.